src/wallet/wallet.h

```diff
@@ -1132,6 +1144,12 @@ class CWallet final : public CCryptoKeyStore, public CValidationInterface
     /* Returns true if HD is enabled */
```

Comment:
`IsHDEnabled()` should be renamed to `HasHDSeed()` because that's actually what it checks (as opposed to `CanSupportFeature(FEATURE_HD)`). Can perhaps wait until a followup, since it's clear enough from the comments.

Reply:
I would prefer to leave it as-is for now. That can be changed in a followup.

Reply:
That's fine, but I'll keep stalking you :-)